components/tikv_util/src/config.rs

```diff
@@ -77,6 +77,10 @@ impl ReadableSize {
     pub const fn as_mb(self) -> u64 {
         self.0 / MB
     }
+
+    pub const fn as_b(self) -> u64 {
```

Comment:
Prefer not to introduce this change in this PR since we used `.0` widely. Better to change all of them at once.